rules/rules-reviewed/openshift/dotnet/db2-unmanaged.windup.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<ruleset id="db2-unmanaged"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a ruleset for .net db2-unmanaged topics when migrating an application to cloud environments.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-base" />
+        </dependencies>
+        <sourceTechnology id="dotnet"/>
+        <targetTechnology id="cloud-readiness"/>
+        <tag>db2-unmanaged</tag>
+    </metadata>
+    <rules>
+        <rule id="db2-unmanaged-0000">
+            <when>
+                <filecontent filename="{*}.{cs|vb}" pattern="IBM.Data.DB2"/>
```

Comment:
We are considering adding multi-language support using https://github.com/konveyor/analyzer-lsp and adding another provider. One of the things that we use is the language server protocol to allow for searching source code based on that language's server. 

I would be interested in if there were some subset of rules that we could work together if you are interested.

Reply:
You can find us on konveyor-dev slack channel in k8s slack or you can email me :) looking forward to hearing from you!

Reply:
@shawn-hurley - Lets connect and see how we can compliment each other.
Please share your contact, I am available at vinod.pahuja@gmail.com